src/parser/shared/parserGenerator.ts

```diff
@@ -201,3 +201,7 @@ export function PointParser(_: any, scanner: DxfArrayScanner) {
 export function ToBoolean({ value }: ScannerGroup) {
 	return !!value;
 }
+
+export function Trim({ value }: ScannerGroup): string {
+    return value.trim();
```

Comment:
There is a possibility of having whitespace in subclass marker. I haven't seen such actual data yet, but I accidently made it when I create the example.